.github/workflows/flake_finder.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Flake Finder
```

Comment:
Do y'all like this job naming? If so, I'll make a parallel name change to the periodic E2E job in the main repo.

Reply:
Sounds appropriate.